src/DateWidget/index.js

```diff
@@ -119,33 +119,33 @@ export default class DateWidget extends React.Component {
       <div className="date-widget-container">
 
         <div className='date-widget-values-container'>
-          <fieldset className='date-fieldset'>
+          <fieldset className='date-fieldset' id={id}>
             <div className="usa-date-of-birth">
               <div className="usa-datefield usa-form-group usa-form-group-month">
-                <label className="input-date-label smaller-input" htmlFor={`${id}Month`}>Month</label>
+                <label className="input-date-label smaller-input" htmlFor={`${id}_Month`}>Month</label>
```

Comment:
isn't JS convention to use camel case as opposed to underscore? super minor quibble

Reply:
Yes! However the ids used by us-form-system are in a css-ish snake case (for example "root_datetime_date"). And when I made that code change I mistakenly thought htmlFor meant more than it did, like it was read by screenreaders or something. Basically it doesn't matter, and my update still doesn't make syntactic sense because I'm capitalizing my suffix. 🤷‍♂️